### DISCUSSION TITLE
Copy Paste Function

### DISCUSSION THREAD

**Discussion by WhatAUsernameIHave** [Deijin27/RanseiLink#58] [Ideas]
A feature that allows you to copy all data from a specific section and paste it to another. For example, copy all of Tackle’s data and overwrite Pound. It would be very useful for making copies of moves, Pokémon, warlord data, etc. The way I imagine it is a Copy button and a Paste button that’ll save the data to a clipboard, instead of using a method like partial transfer. You could also make a clear button that just wipes the data instead of copying. If possible, maybe the clipboard can work between mods so you could copy something from mod A and paste it in mod B.

**Comment by Deijin27** [comment]
This was a feature at one point. I removed it when doing some ui refactoring. If people find it useful, for sure it's easy to add back in for the data-backed things like pokemon, moves, etc.

**Reply by WhatAUsernameIHave** [reply]
To add onto this, a clear-everything option would be nice. Maybe make it so all things are set to 0 and any functions that pull from a table just go to the 000 option.
